assets/js/app.js

```diff
@@ -370,6 +370,23 @@ Hooks.MessageList = {
   }
 };
 
+
+Hooks.RiderSelectionList = {
+  mounted() {
+    this.el.addEventListener("scroll", e => {
+      gap = this.el.scrollHeight - this.el.scrollTop - this.el.clientHeight;
+      console.log("current gap  ", gap);
+      if (gap < 10) {
+        this.pushEventTo(`[id="${this.el.id}"]`, "load_more", {}, (_reply, _ref) => {
+          // do something on event return?
```

Comment:
I don't think we have to adjust the scroll position on this one, so we're good here